py3status/modules/weather_yahoo.py

```diff
@@ -8,11 +8,34 @@
 Find your city code using:
     http://answers.yahoo.com/question/index?qid=20091216132708AAf7o0g
 
+Find your woeid using:
+    http://woeid.rosselliot.co.nz/
+
 Configuration parameters:
     - cache_timeout : how often to check for new forecasts
     - city_code : city code to use
     - forecast_days : how many forecast days you want shown
+    - forecast_format : possible placeholders: {icon}, {low}, {high}, {units},
+        {text}
+    - forecast_include_today: show today forecast as well, default false
```

Comment:
Not really... If `forecast_include_today` is `True`, you get today in the list of forecast, so you still can have in the both today and forecast. In other words you have your current conditions and forecast for today.